package/MDAnalysis/topology/LAMMPSParser.py

```diff
@@ -245,14 +250,14 @@ def read_DATA_timestep(self, n_atoms, TS_class, TS_kwargs):
         positions = np.zeros((n_atoms, 3),
                              dtype=np.float32, order='F')
         try:
-            self._parse_pos(sects['Atoms'], positions)
+            positions = self._parse_pos(sects['Atoms'], positions)
```

Comment:
why the assignment? Can't we then leave/remove the second argument to the function?